h2o-automl/src/test/java/ai/h2o/automl/AutoMLTest.java

```diff
@@ -138,6 +138,7 @@
       assertEquals("wrong amount of standard models", 3, count_non_se);
       assertEquals("wrong amount of SE models", 2, count_se);
       assertEquals(5, aml.leaderboard().getModelCount());
+      Log.info(aml.leaderboard().toTwoDimTable("training_time_millis", "predict_time_per_row_millis"));
```

Comment:
@seb-h2o maybe enum for strings or marker interface with implementations in corresponding columns classes?

Reply:
This shouldn't have been committed, I'll write proper tests. Was just using this to check quickly.
And I won't add enum, each extension column class exposes its name.